Ansible/collection/jfrog/ansible/roles/artifactory_nginx/README.md

```diff
@@ -0,0 +1,5 @@
+# artifactory_nginx
+This role installs NGINX for artifactory. This role is automatically called by the artifactory role and isn't intended to be used separately.
+
+## Role Variables
+* _server_name_: This is the server name. eg. "artifactory.54.175.51.178.xip.io"
```

Comment:
server name sounds like hostname. is this server name or FQDN? 
it says server name, but example is of fqdn